pkg/protostore/protostore.go

```diff
@@ -20,6 +20,8 @@
 // around [pkg.datastore.DataStore] interface and is used specifically to persist Protobuf messages.
 // Just as with [pkg.datastore.DataStore], Protobuf messages can be persisted with revisioning and
 // multi-tenancy support along with `CreatedAt` and `UpdatedAt` timestamps.
+// Tombstone Delete or soft deletes are supported with `DeletedAt` struct field.
+// Use Delete to remove any records that are soft deleted but still in database
```

Comment:
I don't see "do" here @jeyhun

Reply:
Yeah, it's not here anymore. It was there when I last looked